pandas/core/generic.py

```diff
@@ -3086,6 +3087,12 @@ def to_csv(
         encoding : str, optional
             A string representing the encoding to use in the output file,
             defaults to 'utf-8'.
+        errors : str, default 'strict'
+            Specifies how encoding and decoding errors are to be handled.
+            See the errors argument for :func:`open` for a full list
```

Comment:
does this render (open)?

Reply:
It renders as: open()
See https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_hdf.html